package.json

```diff
@@ -27,16 +27,16 @@
     "node": ">=16.16.0"
   },
   "dependencies": {
-    "@visual-regression-tracker/sdk-js": "5.6.0"
+    "@visual-regression-tracker/sdk-js": "5.7.0"
```

Comment:
this is auto incremented

Reply:
What do you mean by it? This version was the latest available at npm registry...

Reply:
not relevant